scripts/setup/ubuntu/install_deps.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+# Copyright Kani Contributors
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+
+set -eu
+
+# Dependencies.
+DEPS=(
```

Comment:
Why do we need all these

Reply:
We can use the docker kani instead.

The plan was to run kani on this modified proptest so that the test cases. However, b/c we got the docker up before this merged, we can use the docker image and avoid installing dependencies.

Reply:
00ca7fd5a46d3733dbf09d49fa8c556f38af1e18